src/mockServer.ts

```diff
@@ -0,0 +1,59 @@
+import { promises as fs } from "fs";
+import * as mockttp from "mockttp";
+import path from "path";
+import selfsigned from "selfsigned";
+
+/*
```

Comment:
Consider moving this comment to be a jsdoc of `startMockServer`, so that the IDE can suggest the usage when using this mock server.